test/lit_tests/diffrules/stablehlo/clamp.mlir

```diff
@@ -0,0 +1,70 @@
+// RUN: enzymexlamlir-opt %s --enzyme-wrap="infn=clamp outfn= retTys=enzyme_dup argTys=enzyme_dup,enzyme_dup,enzyme_dup mode=ForwardMode" --canonicalize --arith-raise | FileCheck %s --check-prefix=FORWARD
+// RUN: enzymexlamlir-opt %s --enzyme-wrap="infn=clamp outfn= retTys=enzyme_active argTys=enzyme_active,enzyme_active,enzyme_active mode=ReverseModeCombined" --canonicalize --remove-unnecessary-enzyme-ops --arith-raise | FileCheck %s --check-prefix=REVERSE
+// RUN: enzymexlamlir-opt %s --enzyme --canonicalize --remove-unnecessary-enzyme-ops --arith-raise | $(dirname $(which enzymexlamlir-opt))/external/stablehlo/stablehlo-translate - --interpret
```

Comment:
I could not find a less hacky way to access `bin/external/stablehlo/stablehlo-translate`